releasenotes/notes/remove-internal-deprecation-module-v1-e38bcb9cfd0603cb.yaml

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
I'm leaning against adding a release note to the PR since it removes an internal module. What do you think?

Reply:
no release note needed since it is already namespaced in `ddtrace.internal`